src/allmydata/storage/immutable.py

```diff
@@ -544,6 +549,9 @@ def advise_corrupt_share(self, reason):
                                             self.shnum,
                                             reason)
 
+    def get_length(self):
+        return self._share_file.get_length()
```

Comment:
and here

Reply:
Ended up deleting this method, per later comment.